config/config.go

```diff
@@ -920,6 +921,8 @@ type MarathonSDConfig struct {
 	Timeout         model.Duration `yaml:"timeout,omitempty"`
 	RefreshInterval model.Duration `yaml:"refresh_interval,omitempty"`
 	TLSConfig       TLSConfig      `yaml:"tls_config,omitempty"`
+	BearerToken     string         `yaml:"bearer_token,omitempty"`
+	BearerTokenFile string         `yaml:"bearer_token_file,omitempty"`
```

Comment:
It's already there in `resolveFilepaths()`:
`   mcfg.BearerTokenFile = join(mcfg.BearerTokenFile)`

It's done the same way as for KubernetesSD:
`    kcfg.BearerTokenFile = join(kcfg.BearerTokenFile)`

I will look into HTTPClientConfig as well.